lib/watirmark_email/qamail.rb

```diff
@@ -1,6 +1,6 @@
 module WatirmarkEmail
   class QAMail < BaseController
-    URL           = "qasendmail.conviocloud.com"
+    URL           = "bvtmail.conviocloud.com"
```

Comment:
shouldn't this be bvtmail.blackbaudcloud.com? Same for the other one?

Reply:
Yes. Updated it.